src/features/applicationMetadata/types.ts

```diff
@@ -6,24 +6,35 @@ type ILogoOptions = {
   size?: 'small' | 'medium' | 'large';
 };
 
-export interface IApplicationMetadata {
-  altinnNugetVersion?: string;
-  createdBy: string;
-  created: string;
-  dataTypes: IDataType[];
+export interface IncomingApplicationMetadata {
   id: string;
-  lastChangedBy: string;
-  lastChanged: string;
+  title: ITitle;
   org: string;
   partyTypesAllowed: IPartyTypesAllowed;
-  title: ITitle;
+  dataTypes: IDataType[];
+  altinnNugetVersion?: string;
   autoDeleteOnProcessEnd: boolean;
   onEntry?: IOnEntry;
   features?: Partial<IBackendFeaturesState>;
   promptForParty?: 'always' | 'never';
   logo?: ILogoOptions;
 }
 
+export type ApplicationMetadata = {
+  isValidVersion: boolean;
+  dataTypes: IDataType[]; // TODO: next step
+  id: string;
+  org: string;
+  partyTypesAllowed: IPartyTypesAllowed; // FIXME: needs improvement
```

Comment:
Which improvements are needed for this one? I'd prefer elaboration or an issue. :pray:

Reply:
Forgot to remove these. Will remove in a later PR 👌